articles/clients/client-settings/native/index.md

```diff
@@ -1,24 +1,20 @@
 ---
-description: Describes the differences between the Auth0 Client types and what they mean when it comes to settings
+description: Client settings for Native
+url: /clients/client-settings/native
+toc: true
 ---
 
-# Client Settings
+# Client Settings: Native
 
 When creating an Auth0 Client, you'll be asked to indicate the *type* of Client you want to create. 
```

Comment:
Maybe instead of this, indicate that this doc lists the client settings for Native apps and they should use the dropdown for a different client type?

Or you could use the `version-warning` instead
::: version-warning
This document lists the settings for Native apps, if....
:::